docs/admin_api/rooms.md

```diff
@@ -478,7 +478,8 @@ The following fields are returned in the JSON response body:
 * `failed_to_kick_users` - An array of users (`user_id`) that that were not kicked.
 * `local_aliases` - An array of strings representing the local aliases that were migrated from
                     the old room to the new.
```

Comment:
Could we note somewhere in this doc that a room can be blocked without the need to join it? Such as:

> * `block` - Optional. If set to `true`, this room will be added to a blocking list, preventing
future attempts to join the room. The server does not need to be in the room in order
to block it. Defaults to `false`.

I realise it's already intuitive that the server would not need to be in the room to block it, but the previous restrictions may have become learned already... and it's best to be explicit?

Reply:
This bit matches the function docstring. I'll keep those two roughly in sync.